src/main/webapp/WEB-INF/views/drivers/add.jsp

```diff
@@ -0,0 +1,14 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Create driver</title>
+</head>
+<body>
+<form method="post" action="${pageContext.request.contextPath}/drivers/add">
+    Model:<input type="text" name="name_driver" required><br>
+    Manufacturer id:<input type="number" name="license_number" required><br>
```

Comment:
fix naming before input tag
you have Model and Manufacturer id but need name and license_number